easybuild/easyconfigs/b/BSMAP/BSMAP-2.74-goolf-1.4.10.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'ConfigureMake' 
+
+name = 'BSMAP'
+version = '2.74'
+
+homepage = 'https://code.google.com/p/bsmap/'
+description = """BSMAP is a short reads mapping software for bisulfite sequencing reads."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['https://bsmap.googlecode.com/files/']
+sources = [SOURCELOWER_TGZ]
+
+skipsteps = ['configure']
+
+installopts = "INSTALL_DIR=%(installdir)s"
+
+patches = [
+     'makefile.patch',
+     'param.patch',
+     'samtools.patch',
```

Comment:
these patch files should be renamed to include the software name and version, and give some clue of what they change/fix

e.g.

```
BSMAP-2.74_samtools-deps.patch
```

Including a line or two of comments on top of the patch file (anything before the first `---` line is comments) would be nice too, next to mentioning the author of the patch.